docs/advanced-features/tabulator.md

```diff
@@ -76,6 +76,35 @@ In addition to the columns defined in the schema, Tabulator will add:
 - `$top_hash` for the revision of the package containing the object (currently
   we query only the `latest` package revision)
 
+### Using Athena to Access Tabulator
+
+Due to the way permissions are configured, Tabulator cannot be accessed from the
+AWS Console or views. You must access Tabulator via the Quilt stack in order to
+query the tables.  This can be done by users via the per-bucket "Queries" tab in
+the Quilt Catalog, or programmatically via `quilt3.login()` and
+`quilt3.get_boto3_session()` to get an Athena client with the same permissions as
+the Quilt stack:
+
+<!--pytest.mark.skip-->
+```python
+import quilt3
+
+quilt3.config('https://YOUR-CATALOG-URL.com/')
+quilt3.login()
+session = quilt3.get_boto3_session()
+athena_client = session.client('athena')
+
+result = athena_client.start_query_execution(
+    QueryString='SELECT * FROM "quilt-YOUR-CATALOG-URL-tabulator"."udp-spec"."ccle-tsv"',
+    QueryExecutionContext={
+        'Database': 'udp-spec'
+    },
+    ResultConfiguration={
+        'OutputLocation': 's3://quilt-YOUR-CATALOG-URL-tabulator-athena-results/'
+    }
+)
```

Comment:
I *think* this won't work because you have to specify workgroup due to our permissions.

Reply:
So... this works, but the full example seems kind of long. Should I move it somewhere else?

Reply:
Moved to the end under "Usage", which I think makes the most sense.